draco/asp_utils.py

```diff
@@ -91,3 +91,19 @@ def blocks_to_program(
         for name, block in blocks.items()
         if isinstance(block, Block) and (keys is None or name in keys)
     ]
+
+
+def get_constants(program: Union[str, Iterable[str]]) -> Dict:
+    if isinstance(program, str):
+        program = program.split(".")
+
+    constants = {}
+    for line in program:
+        match = re.search("#const (.*) = ([-0-9]*)", line)
```

Comment:
Instead of running a regex search, can we use the lingo AST or the lingo API?

Reply:
updated the code using the AST API. 
I'm not sure if this is bad because I'm not familiar with the clingo AST elements and they don't have a lot documentation. So, thanks for the review!